corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -187,14 +187,22 @@ def setUp(self):
         # reset to newly wrapped module
         self.module = self.app.modules[0]
 
-    def test_search_config_model(self, *args):
+    def test_search_config_relevant(self, *args):
         config = CaseSearch()
 
         self.assertEqual(config.get_relevant(), "count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/data/search_case_id]) = 0")  # noqa: E501
 
         config.additional_relevant = "double(now()) mod 2 = 0"
         self.assertEqual(config.get_relevant(), "(count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/data/search_case_id]) = 0) and (double(now()) mod 2 = 0)")  # noqa: E501
 
+    def test_search_config_relevant_multi_select(self, *args):
+        config = CaseSearch()
+
+        self.assertEqual(config.get_relevant(multi_select=True), None)
```

Comment:
Is the `None` return value correctly handled by the callers?

Reply:
Good question, and yes. [This call](https://github.com/dimagi/commcare-hq/blob/669d6ef243dfd69aff3eac870e92f99bb503dd31/corehq/apps/app_manager/suite_xml/post_process/remote_requests.py#L105-L107) checks for None, and the other calls of it ultimately get passed to `get_all_instances_referenced_in_xpaths` which filters out Nones.